deploy/vagrant/create

```diff
@@ -111,7 +118,7 @@ else
     ./vbox/buildbox.sh
   fi
   VUP="vagrant up --provider=$FULL_PROVIDER_NAME --no-provision"
-  [[ "$PROVIDER" == "aws" ]] && VUP="${VUP} --parallel"
+  [[ "$PROVIDER" == "aws" ]] && VUP="${VUP} --parallel" #Provider google could also be added here.
```

Comment:
Only that I'm not sure google-vagrant supports it. It works with or without, so I don't think there is any harm adding it in case 1) it does help and I didn't notice or 2) it eventually gets support. I'll add.